plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestParquetPageSkippingWithOptimizedReader.java

```diff
@@ -26,6 +26,8 @@ protected QueryRunner createQueryRunner()
         return HiveQueryRunner.builder()
                 .setHiveProperties(
                         ImmutableMap.of(
+                                // TODO (https://github.com/trinodb/trino/issues/17397) use optimized writer
```

Comment:
We can point to `https://github.com/trinodb/trino/issues/9359` instead

Reply:
Thanks, updated